src/test/java/seedu/duke/commands/CommandTestUtil.java

```diff
@@ -13,11 +14,11 @@
     public static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d-M-yyyy");
     public static final String MESSAGE_MODULE_ERROR = "There was a problem with getting selected module." + NEWLINE;
 
-    public static ModuleList initialiseModuleList() {
-        ModuleList modules = new ModuleList();
-        Module module = new Module("CS2113T");
-        modules.selectedModule = module;
-        return modules;
+    public static void initialiseModuleList() {
+        TestUtil.removeFiles();
+        ModuleList.loadModuleNames();
+        ModuleList.addModule("CS2113T");
+        ModuleList.setSelectedModule("CS2113T");
```

Comment:
If this is not used anywhere can just delete

Reply:
Ok removed.